lib/web_console/whiny_object.rb

```diff
@@ -0,0 +1,23 @@
+module WebConsole
+  class WhinyObject < SimpleDelegator
```

Comment:
@jmbejar So I got confused here, sorry for that. Dunno why I called that class `Whiny` something to begin with :sweat_smile: I kinda like the logging separation, because I don't like having the request/response objects to log messages, I want most of the methods do be simple queries.

However, this is getting to be a but of an overkill for more than one classes, so I'm okay with your current approach.

I can experiment with separated logging on a separate PR, so no need to bother you with it. I'll take a final look over the PR, but it looks better now.

Thank you for taking the time to work on this.